runner/store.go

```diff
@@ -0,0 +1,101 @@
+package runner
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+type ResultsStore interface {
+	Success(key, result string) error
+	Failure(key, result string) error
+	GetResults(keys []string) ([]string, error)
+}
+
+type DynamoStore struct {
+	client *dynamodb.DynamoDB
+	table  string
+}
+
+func (d DynamoStore) writeResult(success bool, key string, result string) error {
+	_, err := d.client.PutItem(&dynamodb.PutItemInput{
+		Item: map[string]*dynamodb.AttributeValue{
+			"Key": {
+				S: aws.String(key),
+			},
+			"Result": {
+				S: aws.String(result),
```

Comment:
probably want to put a check here for limits around key/val and total item size in dynamo, since this is the raw stdout or stderr